samcli/commands/local/cli_common/invoke_context.py

```diff
@@ -345,6 +349,25 @@ def _clean_running_containers_and_related_resources(self) -> None:
         cast(WarmLambdaRuntime, self.lambda_runtime).clean_running_containers_and_related_resources()
         cast(RefreshableSamFunctionProvider, self._function_provider).stop_observer()
 
+    def _add_account_id_to_global(self):
+        """
+        Attempts to get the Account ID from the current session
+        If there is no current session, the standard parameter override for
+        AWS::AccountId is used
+        """
+        client_provider = samcli.lib.utils.boto_utils.get_boto_client_provider_with_config()
```

Comment:
Could we update the import at the top of this file to import that method instead of qualifying it here?

Reply:
I couldn't get the test to patch correctly if I used `from samcli.lib.utils.boto_utils  import get_boto_client_provider_with_config`. Happy to update if someone can assist here; Python is not my first language..

Reply:
Was the issue that the patch wasn't working when moving the import to the top of the file? You might need to patch it using from the `invoke_context` file instead. That is, instead of
```
@patch("samcli.lib.utils.boto_utils.get_boto_client_provider_with_config")
```
The following patch is used instead
```
@patch("samcli.commands.local.cli_common.invoke_context.get_boto_client_provider_with_config")
```

Reply:
updated the import and the patch as requested.

Reply:
Thanks for the help - I've done the update